docs/integrating-with-unity/defining-states.mdx

```diff
@@ -0,0 +1,334 @@
+---
+sidebar_label: 'Defining States'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Defining States
+
+:::info
+TODO: We need to write a better description about why it is important to limit the state exposed to your bot.
```

Comment:
As I read through this, what comes to mind is the following (not sure how true it is yet though):
- Abstracting away state is important for keep code clean and independent from the rest of your game logic
- It's important to only include the state you need for both efficiency and to make sure bot logic only uses what it can see
- It helps collect state and information from across multiple scripts and game objects into one organized location

Reply:
updated